Movies-MVP/Movies-MVP/Controllers/HomeViewController.swift

```diff
@@ -136,12 +105,13 @@ final class HomeViewController: UIViewController {
     }
 
     private func configureTopRatedButton() {
-        topRatedButton.addTarget(self, action: #selector(showTopRatedAction), for: .touchUpInside)
+        topRatedButton.addTarget(self, action: #selector(showGenreAction), for: .touchUpInside)
         view.addSubview(topRatedButton)
         topRatedButton.setTitle(Constants.MovieTypes.topRated.rawValue, for: .normal)
         topRatedButton.backgroundColor = .systemTeal
         topRatedButton.layer.cornerRadius = 7
```

Comment:
```suggestion
        topRatedButton.layer.cornerRadius = 
```
 и в остальных кнопках тоже